draft-thomson-webpush-protocol.xml

```diff
@@ -804,10 +735,11 @@ DATA         [stream 4] +END_STREAM
        </t>
        <t>
          A subscription URI therefore acts as a bearer token: knowledge of the
-         URI implies authorization to send push messages.  Subscription URIs
-         MUST be extremely difficult to guess.  Encoding a large amount of
-         random entropy (at least 120 bits) in the path component ensures that
-         it is difficult to successfully guess a valid subscription URI.
+         URI implies authorization to either send or receive push messages.
+         Subscription URIs MUST be extremely difficult to guess.  Encoding a
+         large amount of random entropy (at least 120 bits) in the path
+         component ensures that it is difficult to successfully guess a valid
+         subscription URI.
```

Comment:
We need to split this paragraph.  Knowledge of a monitor URI is authorization-enough to listen (and delete).  Knowledge of the push URI is sufficient for sending.